src/cogs/modmail.py

```diff
@@ -0,0 +1,83 @@
+import discord
+from discord.ext import commands
+from discord.ui import Button, View, Modal
+from random import randrange
+import time
+from src import config as conf
+from src.util import permissions
+
+class ModmailModal(Modal):
+    name = discord.ui.TextInput(label = "Subject")
+    issue = discord.ui.TextInput(label = "Content", style = discord.TextStyle.paragraph, max_length = 1000)
+
+    def __init__(self, selected):
+        super().__init__(title = f"Describe your issue")
+        self.selected = selected
+
+    async def on_submit(self, interaction):
+        user = interaction.user
+        channel: discord.TextChannel = interaction.client.get_channel(conf.modmail_channel)
+
+        created_thread = await channel.create_thread(
+            name = f"{self.name.value}",
+            type = discord.ChannelType.private_thread,
+            invitable = False
+        )
+
+        embed = discord.Embed(title = f"**{self.selected}**")
+        embed.add_field(name="", value=f"{self.issue.value}")
+        embed.set_author(name = user.name, icon_url = user.avatar.url)
+
+        await created_thread.add_user(user)
+        await created_thread.send(
+            f"{user.mention} please wait for <@&{conf.staff_role}> to respond.\n",
+            embeds = [embed]
+        )
+
+        await interaction.response.send_message(
+            f"Channel created in <t:{int(time.time())}:f>\n{created_thread.jump_url}", 
```

Comment:
why add time to this message?

Reply:
You requested to show when it was created

Reply:
Did I? I think you might have misunderstood me there :P 

Only the link to the created thread should be relevant here

Reply:
Alr no problem